src/helpers/userHelper.js

```diff
@@ -654,11 +833,31 @@ const userHelper = function () {
               }
             }
           } else if (user?.infringements?.length === 0) {
-            if (moment().diff(moment(user.createdDate), 'months', true) >= elem.months) {
-              if (user.oldinfringements.length === 0 || Math.abs(moment().diff(moment(user.oldinfringements[user.oldinfringements?.length - 1].date), 'months', true)) >= (elem.months - 12)) {
```

Comment:
Note for reviewers:
Here are the only functional/non-ESLint changes
oldinfrigements->oldInfrigements